group_vars/all

```diff
@@ -25,3 +25,10 @@ validate_certs: "yes"
 # env:
 # service_vip:
 # control_interface:
+
+# should mons also run osds
+mon_osd_colocation_allowed: "true"
+
+# OSDs could be run without netplugin on them
+# osd_contiv_network_include controls this behaviour
+osd_contiv_network_include: "true"
```

Comment:
@mapuri yes, I agree that negation is hard to read, but the flip side is that the user needs to specify every capability he wants on the node. 

on the topic of keeping it to storage and not getting down to mon/osd. Is there a case, a host might want to run both ceph and nfs on the same node. wondering to what level should the capability be defined - being able to do nfs based storage is a capability. being able to do ceph storage is another capability. so, do you think specifying "Storage-node" is good enough?